Better Icons/UI

Nobody912 opened the pull request:
Improving the icons that appear under "Window"

w0lfschild committed:
2.5.0

- Bump version number
- Update project settings to recommended
- Add Afloat menubar item if no Window menubar item is found
- Commented out logging

w0lfschild committed:
Updated README.md

- Added info about System Integrity Protection
- Changed EasySIMBL to mySIMBL since EasySIMBL doesn't work anymore and
is no longer maintained
- Removed Chrome info since Chrome is now 64 bit

w0lfschild committed:
Added build

- Added build folder
- Updated .gitignore

w0lfschild committed:
Updated README.md

w0lfschild committed:
Removed Nag

- Removed all of the nag stuff

w0lfschild committed:
2.5.1

- Removed checkForNagOnInstall
- Updated build

w0lfschild committed:
2.5.2

https://github.com/raylee/afloat/commit/680cf27c3f8ecf2603314e1732f18ed7
04924464

w0lfschild committed:
Updated README

w0lfschild committed:
Updated BUILDS

w0lfschild committed:
Updated README

niyue committed:
Fix the download file name the download link

Currently, in the README.md, the download file name (binventory) is incorrect, and the download link leads to a 404 page in github. The issue is fixed by using the correct name (Afloat) and the correct download link (via raw.githubusercontent.com)

w0lfschild committed:
Merge pull request #2 from niyue/fix-download-link

Fix the download link

vjt commented:
Hi, thanks for this. Overall it seems good to me, with the exception of commenting out the `L0Log` statements. I would prefer to keep them, and rather update the log level to not have them enabled by default, but still allowing to enable them when needed. What do you think?

Thank you,

w0lfschild committed:
0.5.3

- Update nib to xib
- Load Afloat menu item outside window menu
- Update project and fix some warnings

w0lfschild committed:
Merge branch 'master' of https://github.com/w0lfschild/afloat

w0lfschild committed:
Update README.md
